src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const DAY_RENT = 40;
+  const BigDiscount = 50;
+  const SmallDiscount = 20;
+
+  if (days < 3) {
+    return days * DAY_RENT;
+  } else if (days >= 3 && days < 7) {
```

Comment:
The condition `days >= 3` is redundant because if the execution reaches this point, `days` is guaranteed to be 3 or more due to the previous `if` check. You can simply check if `days < 7`.